content/docs/faq-ajax.md

```diff
@@ -102,9 +102,9 @@ function MyComponent() {
           setIsLoaded(true);
           setItems(result);
         },
-        // Note: it's important to handle errors here
-        // instead of a catch() block so that we don't swallow
-        // exceptions from actual bugs in components.
+        // 주의: 컴포넌트의 실제 버그에서 발생하는 예외사항들을 넘기지 않도록 
+        // 에러를 catch() 블록(block)에서 처리하기보다는 
+        // 이 부분에서 처리하는 것이 중요합니다.
```

Comment:
```suggestion
        // 주의: 컴포넌트에 있는 실제 버그 때문에 발생한 예외를
        // 처리하지 않도록 catch() 블록 대신에
        // 여기서 에러를 다뤄주는게 중요합니다.
```

"넘긴다"는 표현 대신에 "처리하지 않다"로 바꾸고, 문장을 다듬어서 제안드려봅니다. swallow의 의미를 좀 더 살려보고 싶긴 한데 지금 떠오르는 더 좋은 표현은 없네용 ㅎㅎ

Reply:
상세한 코멘트 감사드립니다.
이 부분에서 '처리하지 않는다'는 표현으로 변경하면서 의미가 바뀐 듯한데, `~발생한 예외를 처리하지 않은 채로 두지 않도록 ~` 으로 수정하는 것은 어떤지요? :)

Reply:
```suggestion
        // 주의: 컴포넌트에 있는 실제 버그 때문에 발생한 예외를
        // 놓치지 않고 처리하기 위해  catch() 블록말고
        // 여기서 에러를 다뤄주는게 중요합니다.
```
`~발생한 예외를 처리하지 않은 채로 두지 않도록 ~`을 swallow의 의미를 살려 `놓치지 않고 처리하기 위해` 로 사용해도 좋을 것 같습니다.
'instead of' 를 대신에 보다 ‘앞말의 대상 이외에’라는 뜻을 나타내는 보조사 '말고' 를 사용하는 것은 어떠신지 제안드려봅니다. :)

Reply:
```suggestion
        // 주의: 컴포넌트에 있는 실제 버그로 인해 발생한 예외를
        // 놓치지 않고 처리하기 위해서는
        // catch() 블록보다는 여기서 에러를 다뤄주는 게 중요합니다.
```
@marvelopers 님이 주신 제안을 좀 더 매끄럽게 다듬어 보았습니다!

Reply:
swallow가 삼킨다로 봐서 "삼키지 마라" -> "처리하지 마라"로 의역했었네요. 근데 "억누르다, 참다" 라는 의미도 있어서 제시해주신 의미가 더 알맞아 보이네용